.github/workflows/clj-build.yml

```diff
@@ -13,7 +13,8 @@ on:
 jobs:
   build:
     name: Build App
-    runs-on: ${{ inputs.runner }}
+    runs-on:
+      group: "build"
```

Comment:
I _think_ this would work, maintaining backwards compatibility while allowing for more complex configuration. I haven't actually tried it yet.
```
runs-on: ${{ fromJSON(inputs.runner) }}
```